docs/reference/contributing/target/sai.md

```diff
@@ -1,24 +1,28 @@
 <h2 id="sai-port">Serial Audio Interface (SAI)</h2>
 
 The **Serial Audio Interface** allows you to send or receive an audio stream over a synchronous serial interface made of 3 to 4 lines.
-- SD : Serial data
-- BCLK : Bit Clock
-- WCLK : Word Clock
-- MCLK : Main Clock (optionnal)
 
-This interface is typically used with Codecs and DACs/ADCs to sample, process and output an audio signal.
+- SD: serial data.
+- BCLK: bit Clock.
+- WCLK: word Clock.
+- MCLK: main Clock (optional).
 
-This is an highly configurable interface where a wide range of element can be adjusted to your need :
-- word length
-- data length (inside words)
-- clocks polarity & phase
-- data alignment...
+A typical use case of this interface is with Codecs and DACs or ADCs to sample, process and output an audio signal.
+
+This highly configurable interface has a wide range of elements you can adjust:
+
+- Word length.
+- Data length (inside words).
+- Clocks polarity and phase.
+- Data alignment.
```

Comment:
This list is not exhaustive, is it worth to signal it here with an ellipsis ?

Reply:
No, it's ok, we are not trying to tell everything just highlight things.